benchexec/tablegenerator/react-table/src/components/SelectColumn.js

```diff
@@ -52,10 +52,32 @@ export default class SelectColumn extends React.Component {
   };
 
   renderColumns = index => {
-    return this.state.list[index].columns.map(column => (
+    const idxRow = [];
+    const columns = this.state.list[index].columns;
+    this.selectable.forEach(function(headerRow, idxHeader) {
+      const found = columns.some(
+        el => el.display_title === headerRow.display_title
+      );
+      if (found) {
+        idxRow.push({
+          display_title: headerRow.display_title,
+          isVisible: columns.find(
+            el => el.display_title === headerRow.display_title
```

Comment:
The `columns.some()` and `columns.find()` with the same callback can be simplified. Just use `find` to get the respective column and replace `if (found)` with `if (column !== undefined)`.

Reply:
Done